platform/mds/mds/src/main/resources/META-INF/motech/mdsContext.xml

```diff
@@ -50,7 +57,7 @@
             <bean factory-bean="mdsConfig" factory-method="getFlywayLocations" />
         </property>
         <property name="sqlMigrationPrefix" value="V"/>
-        <property name="outOfOrder" value="true"/>
+        <property name="outOfOrder" value="false"/>
```

Comment:
Personally, I would keep outOfOrder as true, in case some migrations will have to be introduced to backward releases.